content/en/hosting/monitoring/setup.md

```diff
@@ -141,19 +141,36 @@ With the [release of 1.1.0](https://github.com/medic/cht-watchdog/releases/tag/1
 
 {{% alert title="Note" %}}
 Always run this longer version of the `docker compose` command which specifies both compose files for all future [upgrades](#upgrading).
+This configuration also works for [couch2pg]({{< relref "apps/tutorials/couch2pg-setup" >}} data. Edit `sql_servers.yml` to set the value of `collectors` to `couch2pg`.
 {{% /alert %}}
 
-#### couch2pg Data (Remote)
+#### CHT Sync Data (Remote)
 
-While not the default setup, and not what most deployments need, you may want to set up a way to monitor couch2pg data without sharing any Postgres credentials. Instead of sharing credentials, you expose an HTTP endpoint that requires no login or password.  Of course, similar to  CHT Core's [Monitoring API]({{< relref "apps/reference/api#get-apiv2monitoring" >}}), this endpoint should be configured to not share sensitive information (since it will be publicly accessible).
+While not the default setup, and not what most deployments need, you may want to set up a way to monitor CHT Sync data without sharing any Postgres credentials. Instead of sharing credentials, you expose an HTTP endpoint that requires no login or password.  Of course, similar to  CHT Core's [Monitoring API]({{< relref "apps/reference/api#get-apiv2monitoring" >}}), this endpoint should be configured to not share sensitive information (since it will be publicly accessible).
```

Comment:
```suggestion
While not the default setup, and not what most deployments need, you may want to set up a way to monitor CHT Sync or couch2pg data without sharing any Postgres credentials. Instead of sharing credentials, you expose an HTTP endpoint that requires no login or password.  Of course, similar to  CHT Core's [Monitoring API]({{< relref "apps/reference/api#get-apiv2monitoring" >}}), this endpoint should be configured to not share sensitive information (since it will be publicly accessible).
```